pkg/kademlia/replacement_cache_test.go

```diff
@@ -33,3 +36,25 @@ func TestAddToReplacementCache(t *testing.T) {
 	rt.addToReplacementCache(kadBucketID2, node4)
 	assert.Equal(t, []*pb.Node{node3, node4}, rt.replacementCache[kadBucketID2])
 }
+
+func TestRemoveFromReplacementCache(t *testing.T) {
+	ctx := testcontext.New(t)
+	defer ctx.Cleanup()
+	rt := createRoutingTableWith(storj.NodeID{244, 255}, routingTableOpts{cacheSize: 3})
+	defer ctx.Check(rt.Close)
+
+	kadBucketID2 := bucketID{127, 255}
+	node2 := teststorj.MockNode(string([]byte{100, 255}))
```

Comment:
Use `node2 := storj.NodeID{100, 255}`

Reply:
i need the actual node here though

Reply:
ah, sorry... mistakenly read it was only creating node id